library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -12,6 +12,87 @@ typedef struct {
     libspdm_get_spdm_response_func get_response_func;
 } libspdm_get_response_struct_t;
 
+typedef struct {
+    uint8_t code;
+    const char *code_str;
+} libspdm_code_str_struct_t;
+
+/**
+ * Return the GET_SPDM_RESPONSE function via last request.
+ *
+ * @param  spdm_context                  The SPDM context for the device.
+ *
+ * @return GET_SPDM_RESPONSE function according to the last request.
+ **/
+const static char *libspdm_get_code_str(uint8_t code)
```

Comment:
This needs to be wrapped in `LIBSPDM_DEBUG_PRINT_ENABLE` so that it's not present in release code.

Reply:
thanks, fixing it